lambda/step-lambda.js

```diff
@@ -17,29 +17,46 @@ const documentClient = new AWS.DynamoDB.DocumentClient({'region': region})
 // result is the returned proomised object (awaits until async function completed) - i promise rejected an exception is thrown
 // NB 'context' parameter has info in it tthat we could find useful and extract, although in this example we don't need it
 
-exports.handler = async function(event, context) {
+exports.handler = function(event, context) {
```

Comment:
I suspect you'll want to keep this as an async function as it'll be easier to deal with the promises.